concrete/controllers/panel/detail/page/composer.php

```diff
@@ -60,6 +60,21 @@ public function saveAndExit()
         }
     }
 
+    public function justSave()
+    {
+        if ($this->validateAction()) {
+            $r = $this->save();
```

Comment:
Can you use full variable names in core code to improve readability? `$result` instead of `$r` etc

Reply:
If you'd like, I just want to make sure new code matches guidelines.

Reply:
@KorvinSzanto Thanks. Where I can read the guidelines?

Reply:
You can see guidelines here: https://documentation.concrete5.org/developers/background/coding-style-guidelines